asv_bench/benchmarks/pandas/frame_ctor.py

```diff
@@ -0,0 +1,225 @@
+import numpy as np
+
+import modin.pandas as pd
+from modin.pandas import (
```

Comment:
We may not want to test this separately given Modin lazy execution abilities, @anmyachev thoughts?

Reply:
I'm not sure if the current constructors are lazily executed. Therefore, it is probably interesting to see an Modin' overhead in the construction of objects.

Reply:
Do you think we need all of these cases?